modules/installation-overview.adoc

```diff
@@ -179,7 +179,7 @@ the cluster to the infrastructure that you provided.
 If you do not use infrastructure that the installation program provisioned, you must manage
 and maintain the cluster resources yourself, including:
 
-* The control plane and compute machines that make up the cluster
+* The underlying compute infrastructure
```

Comment:
@sreber84, will you PTAL at this suggested change?

```suggestion
* The hosts for the control plane and compute machines that make up the cluster
```

If you're ok with that wording, I'm happy to merge it.